tests/integration/conftest.py

```diff
@@ -8,7 +8,10 @@
 def setup_server():
     "Start server"
     install_cmds = [
-
+        ["wget", "-qO", "-", "https://deb.edgepi.com/edgepi-release.gpg", "|",
+         "sudo", "gpg", "--dearmor", "-o", "/usr/share/keyrings/edgepi-release.gpg"],
+        ["sudo", "tee", "/etc/apt/sources.list.d/edgepi.list", ">/dev/null"],
```

Comment:
that command is not correct. this is the command you need to run:
```
sudo tee -a /etc/apt/sources.list.d/edgepi.list >/dev/null <<'EOF'
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/main bullseye main
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/test bullseye test
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/nightly bullseye nightly
EOF
```

Reply:
Could you take a look at the conftest.py code now? The edgepi-rpc-server still isn't located but I don't see any errors in my code.

Reply:
Could you take another look at the file? The edgepi-rpc-server package still isn't located but I don't see any errors in my code.